cli/data.py

```diff
@@ -175,13 +179,18 @@ def aggregate_cbsa(output_path: pathlib.Path):
 
 
 @main.command()
-@click.argument("output_path", type=pathlib.Path)
-def aggregate_states_to_country(output_path: pathlib.Path):
-    us_timeseries = combined_datasets.load_us_timeseries_dataset()
-    country_dataset = region_aggregation.aggregate_regions(
-        us_timeseries, pipeline.us_states_to_country_map(),
+def aggregate_states_to_country():
+    dataset = timeseries.MultiRegionDataset.from_wide_dates_csv(
+        pathlib.Path("data/pre-agg-wide-dates.csv")
+    ).add_static_csv_file(pathlib.Path("data/pre-agg-static.csv"))
+    dataset = region_aggregation.aggregate_regions(
+        dataset,
+        pipeline.us_states_and_territories_to_country_map(),
+        reporting_ratio_required_to_aggregate=DEFAULT_REPORTING_RATIO,
+    )
+    dataset.write_to_wide_dates_csv(
+        pathlib.Path("data/post-agg-wide-dates.csv"), pathlib.Path("data/post-agg-static.csv")
```

Comment:
Is this code useless without the commented-out code above? Seems like it at least warrants a TODO comment explaining the state of affairs and how / when you'd use this command.

Reply:
I added a `help` attribute with some info so it appears in `./run.py data --help` and `./run.py data aggregate-states-to-country --help` and the source.